core/templates/dev/head/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -63,26 +63,30 @@ oppia.factory('ExplorationEngineService', [
   'EditableExplorationBackendApiService', 'ExplorationHtmlFormatterService',
   'ExplorationObjectFactory', 'ExpressionInterpolationService',
   'FocusManagerService', 'GuestCollectionProgressService',
-  'ImagePreloaderService', 'LanguageUtilService', 'LearnerParamsService',
-  'NumberAttemptsService', 'PlayerCorrectnessFeedbackEnabledService',
-  'PlayerTranscriptService', 'ReadOnlyExplorationBackendApiService',
-  'StateCardObjectFactory', 'StateClassifierMappingService',
-  'StatsReportingService', 'UrlInterpolationService', 'UserService',
-  'WindowDimensionsService', 'DEFAULT_PROFILE_IMAGE_PATH', 'INTERACTION_SPECS',
-  'PAGE_CONTEXT', 'WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS',
+  'ImagePreloaderService', 'LanguageUtilService',
+  'LearnerParamsService', 'NumberAttemptsService',
+  'PlayerCorrectnessFeedbackEnabledService', 'PlayerTranscriptService',
+  'ReadOnlyExplorationBackendApiService', 'StateCardObjectFactory',
+  'StateClassifierMappingService', 'StatsReportingService',
+  'UrlInterpolationService', 'UserService', 'WindowDimensionsService',
+  'DEFAULT_PROFILE_IMAGE_PATH', 'ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE',
+  'INTERACTION_SPECS', 'PAGE_CONTEXT',
+  'WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS',
   function(
       $http, $q, $rootScope, AlertsService, AnswerClassificationService,
       AudioPreloaderService, AudioTranslationLanguageService, ContextService,
       EditableExplorationBackendApiService, ExplorationHtmlFormatterService,
       ExplorationObjectFactory, ExpressionInterpolationService,
       FocusManagerService, GuestCollectionProgressService,
-      ImagePreloaderService, LanguageUtilService, LearnerParamsService,
-      NumberAttemptsService, PlayerCorrectnessFeedbackEnabledService,
-      PlayerTranscriptService, ReadOnlyExplorationBackendApiService,
-      StateCardObjectFactory, StateClassifierMappingService,
-      StatsReportingService, UrlInterpolationService, UserService,
-      WindowDimensionsService, DEFAULT_PROFILE_IMAGE_PATH, INTERACTION_SPECS,
-      PAGE_CONTEXT, WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS) {
+      ImagePreloaderService, LanguageUtilService,
+      LearnerParamsService, NumberAttemptsService,
+      PlayerCorrectnessFeedbackEnabledService, PlayerTranscriptService,
+      ReadOnlyExplorationBackendApiService, StateCardObjectFactory,
+      StateClassifierMappingService, StatsReportingService,
+      UrlInterpolationService, UserService, WindowDimensionsService,
+      DEFAULT_PROFILE_IMAGE_PATH, ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE,
+      INTERACTION_SPECS, PAGE_CONTEXT,
+      WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS) {
```

Comment:
is this change required?

Reply:
Reverted

Reply:
What exactly was reverted?

Reply:
Removing ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE